### DISCUSSION TITLE
NixOS Support

### DISCUSSION THREAD

**Discussion by timkoval** [ruyrybeyro/chrootvpn#20] [Q&A]
I've tried the latest PR with NixOS Support but unfortunately it doesn't work properly with chroot (cannot find the command)
Could you please help me to figure it out if you have any ideas? Thank you! @robsonrod @ruyrybeyro

**Comment by ruyrybeyro** [comment]
Although they are installed, debootstrap seems to not be able to find/have hardcoded PATHs, to at least chroot, grep, head, sed and cut. Have not find a workaround for it yet.
